src/monmove.cpp

```diff
@@ -1475,11 +1475,19 @@ bool monster::move_to( const tripoint &p, bool force, const float stagger_adjust
 
     if( has_flag( MF_DRIPS_NAPALM ) ) {
         if( one_in( 10 ) ) {
-            g->m.add_item_or_charges( pos(), item( "napalm" ) );
+            // if it has more napalm, drop some and reduce ammo in tank
+            if( ammo["pressurized_tank"] > 0 ) {
```

Comment:
Does this need an "isset" check for a case where the `MF_DRIPS_NAPALM` flag is added to a monster without giving that monster the right ammo type?